src/Microsoft.Data/PrimitiveDataFrameColumnContainer.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Data
+{
+    public partial class PrimitiveDataFrameColumnContainer<T>
```

Comment:
From a public (or a user's) perspective, what is the difference between PrimitiveDataFrameColumn and PrimitiveDataFrameColumnContainer?

Reply:
A `PrimitiveDataFrameColumnContainer` is just an internal store(changed the modifier to `internal` now) to hold the column data. `PrimitiveDataFrameColumn` defines the APIs that manipulate the container